virttest/qemu_devices/qcontainer.py

```diff
@@ -191,7 +191,7 @@ def qemu_version(self):
         return self.__qemu_ver
 
     def initialize_iothread_manager(self, params, guestcpuinfo,
-                                    cmdline_format_cfg=None):
+                                    cmdline_format_cfg={}):
```

Comment:
Although it's safe to use {} (read-only) as the default  here, I still suggest using None, and then we set it {} if cmdline_format_cfg is None.

Up to U, it's OK for me too.

Reply:
> Although it's safe to use {} (read-only) as the default here, I still suggest using None, and then we set it {} if cmdline_format_cfg is None.
> 
> Up to U, it's OK for me too.

Hi @zhencliu , if we use None at here, we have to check None and change None to {}. I prefer to set {} at here.
But, thanks your comment.